### PR TITLE
move label to avoid calling multi_patient_finder; prettify dated_remi…

### LINKED ISSUE #4105: No need for patient name window to pop up
### Describe the bug

When I am in the message center, looking at a message. I click on the blue word “Patient:” which is supposed to open that patient’s chart. Well it does open that patients chart but it also pops up a Patient name: window. I now have to close this window to proceed.

### Expected behavior
 It should go to the patient chart without popping up a Patient Name Window.

### Client configuration

- Browser:  Chrome and Firefox
- OpenEMR version: 5.0.2(5)
- Windows and Ubuntu

### PR DESCRIPTION
**Pull request by stephenwaite**
…nders.php

Fixes #4105 

#### Short description of what this resolves:
stops `a` tag inside label from firing `multi_patient_finder()` when trying to navigate to pt dashboard

#### Changes proposed in this pull request:
moved `label`, quasi psr 12 styling to `dated_reminders.php`

### CODE REVIEW

**Comment by stephenwaite** [comment]
testing well, sorry for all the meaningless styling @bradymiller , the only code change is the move of the label element

File: interface/main/messages/messages.php [hunk 1]
```diff
@@ -393,30 +393,27 @@ class='form-horizontal'
                                                     generate_form_field(array('data_type' => 1, 'field_id' => 'message_status', 'list_id' => 'message_status', 'empty_title' => 'SKIP', 'order_by' => 'title', 'class' => 'form-control'), $form_message_status); ?>
                                                 </div>
                                                 <div class="col-6 col-md-4">
-                                                    <label for="form_patient">
-                                                        <?php
+                                                    <?php
                                                         if ($task != "addnew" && $result['pid'] != 0) { ?>
-                                                            <a class="patLink" onclick="goPid('<?php echo attr(addslashes($result['pid'])); ?>')"><?php echo xlt('Patient'); ?>:</a>
+                                                            <a class="patLink" onclick="goPid('<?php echo attr(addslashes($result['pid'])); ?>')" title='<?php echo attr(xlt('Click me to Open Patient Dashboard')) ?>'><?php echo xlt('Patient'); ?>:</a><label for="form_patient">&nbsp</label>
```
Comment (bradymiller) [inline interface/main/messages/messages.php:398]: around the new title variable: remove the attr() and change xlt() to xla()
(escaping fun :) )

**Comment by bradymiller** [comment]
looks good (after get it to pass styling testing). thanks for the fix!
(btw, don't ever worry about the styling work since I can filter out whitespace stuff with `?w=1` at end of address `https://github.com/openemr/openemr/pull/4430/files?w=1` :) )

**Comment by stephenwaite** [comment]
bringing in, thanks @bradymiller !

**Pull request merged**